Makefile

```diff
@@ -10,11 +10,14 @@ cumulus:
 test:
 	go test $(PACKAGES) --cover
 
+run-console:
```

Comment:
You can specify a dependency for `run-console` like
```
run-console: cumulus
```
so that cumulus must be up-to-date before we run the `run-console` target.

Might make this code cleaner